### DISCUSSION TITLE
PagerDuty plugin

### DISCUSSION THREAD

**Discussion by sonicaghi** [Kong/kong#7384] [Ideas and feature requests]
pagerduty can trigger sms/phone alerts based on http status codes. for example a pagerduty customers (and kong user) can setup SMS alerts once Kong receives 5XX > 10

**Comment by derDieDasJojo** [comment]
I think an Twilio integration would be even more comfortable.
then you could send the allerting sms very cheap instead of paying a monthly bill.
